submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/repositories/Repositories.kt

```diff
@@ -36,6 +36,10 @@ interface SubmissionDataRepository : JpaRepository<Submission, Long> {
     fun getFirstByAccNoOrderByVersionDesc(accNo: String): Submission
 
     fun existsByAccNo(accNo: String): Boolean
+
+    fun findDistinctByRootSectionTypeAndAccessTagsInAndVersionGreaterThan(
```

Comment:
Two comments about this query.

1. Usually, when is that long I rather prefer use `@query`
2. As accessionNo of project is the same of its access tags we can simplify query.

```
    @Query("Select sub From Submission sub, Section se Where " +
        "sub.version > 0 and sub.rootSection = se and se.type = :type and sub.accNo in (:accessTags)")
    fun findByTypeAndAccNo(type: String, accessTags: List<String>): List<Submission>
```

and then...
```
fun findProjectsByAccessTags(tags: List<AccessTag>) = 
submissionRepository.findByTypeAndAccNo(Project.value, tags.map { it.name }).map { submissionDbMapper.toSubmission(it) }```